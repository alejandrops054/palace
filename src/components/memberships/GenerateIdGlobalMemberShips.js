/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import {React, useState} from 'react';
import {Input, Select, Checkbox} from "../index";
import SeachNav from "../ui/search"
import TextField from '@material-ui/core/TextField';
import Modal from 'react-bootstrap/Modal'
import Button from 'react-bootstrap/Button'

const GenerateIdGlobalMemberShips = ()=>{

    const Titulo =[
        'Generate Id GlobalMemberShips'
    ]

    const [show, setShow] = useState(false);

    const handleClose = () => setShow(false);
    const handleShow = () => setShow(true);

    return (
        <>
        <Button variant="primary" onClick={handleShow}>
          {Titulo}
        </Button>
  
        <Modal show={show} onHide={handleClose} size="ms" aria-labelledby="contained-modal-title-vcenter" centered>
          <Modal.Header closeButton>
            <Modal.Title>{Titulo}</Modal.Title>
          </Modal.Header>
          <Modal.Body>
              <>
              <from>
                <div className="container">
                    <div className="row">
                        <div className="col-md-4">
                            <Checkbox name="checkRow3" label="For Application" />
                        </div>
                        <div className="col-md-4">
                            <Checkbox name="checkRow3" label="For Queantity" /> 
                        </div>
                        <div className="col-md-4">
                            <Checkbox name="checkRow3" label="All" />
                        </div>
                        <div className="col-md-12">
                            <Input
                                type="text"
                                label=""
                                placeholder=""
                                col={12}
                            />
                        </div>
                    </div>
                </div>
              </from>
              </>
          </Modal.Body>
          <Modal.Footer>

            <Button variant="primary">
              Generate
            </Button>
          </Modal.Footer>
        </Modal>
        </>
    )
}

export default GenerateIdGlobalMemberShips