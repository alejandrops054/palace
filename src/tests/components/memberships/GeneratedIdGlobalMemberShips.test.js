/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import GenerateIdGlobalMemberShips from '../../../components/memberships/GenerateIdGlobalMemberShips'

describe('GenerateIdGlobalMemberShips component',() => {
    test('debe mostrar snapshot', () => {
        
        const wrapper = shallow(<GenerateIdGlobalMemberShips/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})