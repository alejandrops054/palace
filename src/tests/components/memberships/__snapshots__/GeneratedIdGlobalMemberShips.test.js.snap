// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenerateIdGlobalMemberShips component debe mostrar snapshot 1`] = `
<Fragment>
  <Button
    active={false}
    disabled={false}
    onClick={[Function]}
    variant="primary"
  >
    Generate Id GlobalMemberShips
  </Button>
  <Modal
    animation={true}
    aria-labelledby="contained-modal-title-vcenter"
    autoFocus={true}
    backdrop={true}
    centered={true}
    dialogAs={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "render": [Function],
      }
    }
    enforceFocus={true}
    keyboard={true}
    onHide={[Function]}
    restoreFocus={true}
    show={false}
    size="ms"
  >
    <ModalHeader
      closeButton={true}
      closeLabel="Close"
    >
      <ModalTitle>
        Generate Id GlobalMemberShips
      </ModalTitle>
    </ModalHeader>
    <ModalBody>
      <from>
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-4"
            >
              <Checkbox
                label="For Application"
                name="checkRow3"
              />
            </div>
            <div
              className="col-md-4"
            >
              <Checkbox
                label="For Queantity"
                name="checkRow3"
              />
            </div>
            <div
              className="col-md-4"
            >
              <Checkbox
                label="All"
                name="checkRow3"
              />
            </div>
            <div
              className="col-md-12"
            >
              <Input
                col={12}
                label=""
                placeholder=""
                type="text"
              />
            </div>
          </div>
        </div>
      </from>
    </ModalBody>
    <ModalFooter>
      <Button
        active={false}
        disabled={false}
        variant="primary"
      >
        Generate
      </Button>
    </ModalFooter>
  </Modal>
</Fragment>
`;
